pyquil/api/_qvm.py

```diff
@@ -128,7 +127,7 @@ def connect(self) -> None:
         try:
             version_dict = self.get_version_info()
             check_qvm_version(version_dict)
-        except ConnectionError:
+        except RequestError:
```

Comment:
No more `requests` module, due to using `httpx` transitively, so replaced this with the nearest